packages/sdks/src/blocks/slot/slot.lite.tsx

```diff
@@ -0,0 +1,49 @@
+import type { Signal } from '@builder.io/mitosis';
+import { useMetadata, useStore } from '@builder.io/mitosis';
+import Blocks from '../../components/blocks/blocks.lite.jsx';
+import type { BuilderContextInterface } from '../../context/types.js';
+import type { BuilderBlock } from '../../types/builder-block.js';
+
+export interface DropzoneProps {
+  name: string;
+  builderBlock: BuilderBlock;
+  builderContext: Signal<BuilderContextInterface>;
+  attributes: any;
+}
+
+useMetadata({
+  rsc: {
+    componentType: 'server',
+  },
+});
+
+export default function Slot(props: DropzoneProps) {
+  const state = useStore({
+    get symbolId() {
+      return props.builderContext.value.context?.symbolId as string | undefined;
+    },
+    get blocks() {
+      return (
+        (props.builderContext.value.rootState[props.name] as BuilderBlock[]) ||
+        []
+      );
+    },
+  });
+  return (
+    <div
+      style={{
+        pointerEvents: 'auto',
+      }}
+      {...(!state.symbolId && {
+        'builder-slot': props.name,
+      })}
+    >
+      <Blocks
+        parent={state.symbolId}
```

Comment:
I would inline the `symbolId` instead of creating an extra piece of state. It's generally safer that way, given how reactivity is different for each web framework

Reply:
True, I've made them inline